apps/rush/src/actions/ChangeAction.ts

```diff
@@ -334,7 +333,7 @@ export default class ChangeAction extends BaseRushAction {
   private _writeChangeFile(changeFile: IChangeFile): Promise<void> {
     const output: string = JSON.stringify(changeFile, undefined, 2);
 
-    const filePath: string = this._generateChangeFilePath(changeFile.packageName);
+    const filePath: string = _generateChangeFilePath(this.rushConfiguration, changeFile.packageName);
```

Comment:
Rush is not writing change files in sync. It cannot use that method.

---
In reply to: [126511647](https://github.com/Microsoft/web-build-tools/pull/267#discussion_r126511647) [](ancestors = 126511647)